src/components/Attachments/AttachmentCarousel/index.tsx

```diff
@@ -88,10 +89,16 @@ function AttachmentCarousel({report, reportActions, parentReportActions, source,
             return;
         }
 
-        const initialPage = targetAttachments.findIndex(compareImage);
+        let initialPage = targetAttachments.findIndex(compareImage);
+        const currentPage = attachments.findIndex(compareImage);
```

Comment:
```suggestion
        const prevInitialPage = attachments.findIndex(compareImage);
```

I think `prevInitialPage` is more clear as it indicates it's just another "initialPage".